Hello Microverse pull request

abiy006 opened the pull request:
Tasks accomplished on this PR

- Created a test.md
- Added test.md to gitignore
- Added node_modules to gitignore
- Set up linters
- Customizing README.md file
- Added index.html
- Added style.css
- Adding a license to a repository

Shedrack-Sunday commented on README.md:
````diff
@@ -1 +1,289 @@
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+<!--
+Example command:
+
+```sh
+ gem install rails
+```
+ -->
+
+### Setup
+
+Clone this repository to your desired folder:
+
+<!--
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/my-project.git
+```
+--->
````
- [ ] Please fill up these sections to direct users on how to touse your project. 
- The comments are there as a guide. replace it by customizing it to your app 👍

Shedrack-Sunday commented on README.md:
````diff
@@ -1 +1,289 @@
+```
+--->
+
+### Install
+
+Install this project with:
+
+<!--
+Example command:
+
+```sh
+  cd my-project
+  gem install
+```
+--->
+
+### Usage
+
+To run the project, execute the following command:
+
+<!--
+Example command:
+
+```sh
+  rails server
````
- [ ] As you have linters in your app, edit the install section to show users how to install all dependencies needed
- [ ] For the usage, show steps on how to view th index.html file on the live server since its not deployed yet

Shedrack-Sunday commented on README.md:
````diff
@@ -1 +1,289 @@
+```
+--->
+
+### Run tests
+
+To run tests, run the following command:
+
+<!--
+Example command:
+
+```sh
+  bin/rails test test/models/article_test.rb
+```
+--->
+
+### Deployment
+
+You can deploy this project using:
+
+<!--
+Example:
+
+```sh
+
+```
+ -->
````
-   [ ] You can delete this section since we don't have tests and deployment on the project yet 👍

Shedrack-Sunday commented on README.md:
```diff
@@ -1 +1,289 @@
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+> Mention all of the collaborators of this project.
+
```
- [ ] Kindly delete this section of the README, as its placed to give you a guide but should not be added to your project 👍

Shedrack-Sunday commented on README.md:
```diff
@@ -1 +1,289 @@
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+👤 **Author2**
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
```
- [ ] You're the only contributor to the project. delete the second author, them fill up the first with our details 👍

Shedrack-Sunday commented on README.md:
```diff
@@ -1 +1,289 @@
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
```
- [ ] Kindly delete this section of the README, as its placed to give you a guide but should not be added to your project 👍

Shedrack-Sunday commented on README.md:
```diff
@@ -1 +1,289 @@
+## 🔭 Future Features <a name="future-features"></a>
+
+> Describe 1 - 3 features you will add to the project.
+
+- [ ] **[new_feature_1]**
+- [ ] **[new_feature_2]**
+- [ ] **[new_feature_3]**
```
- [ ] The project obviously has room for future improvements. Kindly specify three of such here 👍

Shedrack-Sunday commented on README.md:
```diff
@@ -1 +1,289 @@
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
```
- [ ] Kindly delete this section of the README, as its placed to give you a guide but should not be added to your project 👍

Shedrack-Sunday commented on README.md:
```diff
@@ -1 +1,289 @@
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+> Give credit to everyone who inspired your codebase.
+
+I would like to thank...
```
- [ ] For this section, first delete the comments which serve as a guide on line `259`, then complete the acknowledgment part. e.g to microverse for inspiring th project 👍

Shedrack-Sunday commented on README.md:
```diff
@@ -1 +1,289 @@
+
+<!-- FAQ (optional) -->
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+> Add at least 2 questions new developers would ask when they decide to use your project.
+
+- **[Question_1]**
+
+  - [Answer_1]
+
+- **[Question_2]**
+
+  - [Answer_2]
```
- [  ] This is the only optional portion of your README. either you decide to fill it up with potential questions users will have, or you delete the section 👍

Shedrack-Sunday commented on README.md:
```diff
@@ -1 +1,289 @@
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```
- [ ]This is a comment suggestion, and should not be included in your code to make it professional. Kindly omit it

Shedrack-Sunday reviewed (changes requested):
Hi @abiy006    👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well implementing the requirements,  However, there are a few you need to work on, to move to the next project. 💯

 
![](https://user-images.githubusercontent.com/80612925/157397398-00904e26-60ec-46cd-9510-240661c250ef.gif)


Highlights
✅All linters are checked
✅ Descriptive commit messages 
✅ Moderate design 
  
Required Changes ♻️

- [x] You did a great job with implementing the project requirements, however, there are a few things I suggest you change and fill up concerning your README  

- [x] Kindly  make your pull request summary more desriptive 

## View:
![image](https://user-images.githubusercontent.com/50990925/219523997-dd0ef7d3-9c8e-4419-ad9e-b7723184a55c.png)


_check the comment section for a detailed review_ 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

GedeonTS commented on README.md:
```diff
@@ -256,34 +125,6 @@ If you like this project...
-
-## 📝 License <a name="license"></a>
-
-This project is [MIT](./LICENSE) licensed.
```
-  [ ] Good job deleting the unnecessary lines as suggested by the previous reviewer, However, in the process you have removed the License section, kindly bring it back :recycle: 
-

GedeonTS reviewed (changes requested):
# STATUS: CHANGES REQUIRED :recycle: :
Hi @abiy006 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

<img src="https://media.giphy.com/media/31Q36EHJ7C5ZoIMw79/giphy.gif">

 ## Highlights :tophat: :+1: 
- No linter errors :+1: 
- Good use of Github flow and gitignore :+1: 

 ## Required Changes ♻️
- As requested by the previous reviewer, kindly make sure to add more details to the PR, by mentioning for example:
 - Created a test.md
 - Added test.md to gitignore 
 - Set up linters
 - ....
_Check the comments under the review._

## Optional suggestions :ringed_planet: 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

wayungi reviewed (changes requested):
Hi @abiy006 , 

![image](https://user-images.githubusercontent.com/99656687/219614539-dcd328d7-b1f4-423c-9d7d-2fa172aaa07c.png)

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
- [ ] All linter checks passing ✔️ 
- [ ] No merge conflicts ✔️ 
- [ ]  Correct GitHub flow ✔️ 

 ## Required Changes ♻️
_Check the comments under the review._
- [ ] Your PR is not descriptive enough. Previous code reviewers have given you good guidance. Please put it into practice. 👍🏾 
![pr](https://user-images.githubusercontent.com/99656687/219613616-1bd1b1bd-8c43-4cec-b5ed-889794d2bdfe.png)
`Please follow the structure below 👇🏾 
**In this PR, I have added the following**
  - Created a test.md
  - Added test.md to gitignore
  - Set up linters
`
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

leonardodiasb reviewed (approved):
# Status: ✔️ Project Approved ✔️ 

Hello @abiy006 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 All linter checks passing ✔️
 No merge conflicts ✔️
 Correct GitHub flow ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

abiy006 commented:
Tasks accomplished on this Merge PR

- Created a test.md
- Added test.md to gitignore
- Added node_modules to gitignore
- Set up linters
- Customizing README.md file
- Added index.html
- Added style.css
- Adding a license to a repository

Ibizugbe commented on Readme.md:
```diff
@@ -1 +1,158 @@
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
```
Nice work on your readme👍, Kindly ensure every section of the readme is filled and personalized for this project😉
